examples/meta/src/multiclass/ecoc_discriminant_aed.sg.in

```diff
@@ -0,0 +1,40 @@
+CSVFile f_feats_train("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("@SHOGUN_DATA@/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+CSVFile f_labels_test("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_test.dat")
```

Comment:
`CSVFile` has now been removed from the interfaces, you now need to use `File` and `read_csv` instead. Have a look at the other examples

Reply:
Yes okay, I'll do that. Are there any other changes to be made as well?

Reply:
You'll have to see!